diff --git a/ftdetect/ts.vim b/ftdetect/ts.vim
deleted file mode 100644
index 07a3dc7..0000000
--- a/ftdetect/ts.vim
+++ /dev/null
@@ -1 +0,0 @@
-autocmd BufNewFile,BufRead *.ts setfiletype xml
diff --git a/plugin/synmenu.vim b/plugin/synmenu.vim
deleted file mode 100644
index 551e213..0000000
--- a/plugin/synmenu.vim
+++ /dev/null
@@ -1,3 +0,0 @@
-if has("menu") && &guioptions !~# "M"
-  runtime! synmenu.vim
-endif
